### PR TITLE
Add makie quote and more blue code compliances.

### PR DESCRIPTION
**Pull request by lazarusA**
Just small fixes

### CODE REVIEW

**Review by rikhuijzer — APPROVED** [review]

File: contents/dataVis.md [hunk 1]
```diff
@@ -56,15 +56,18 @@ Plotly, PGFPlotsX, PyPlot?
...
 - corrplot and cornerplot
 
-## `Makie.jl` {#sec:makie}
+## Makie.jl {#sec:makie}
 
-(WIP) ideas for a better openinig line? 
+> From the japanese word Maki-e, which is a technique to sprinkle lacquer with gold and silver powder.
+> Data is the gold and silver of our age, so let's spread it out beautifully on the screen!
+>
+> _Simon Danisch_
```
Comment (storopoli) [inline contents/dataVis.md:64]: Maybe add who Simon Danisch is?

Something like "creator of Makie.jl"

File: contents/dataVis.md [hunk 2]
```diff
@@ -56,15 +56,18 @@ Plotly, PGFPlotsX, PyPlot?
...
 
 [Makie.jl](http://makie.juliaplots.org/stable/index.html#Welcome-to-Makie!) is a high-performance, extendable, and multi-platform plotting ecosystem for the Julia programming language.
 In our opinion, it is the prettiest and most versatile plotting package.
 
 `Makie.jl` is the frontend package that defines all plotting functions.
-It is reexported by every backend, so you don't have to specifically install or import it.
+It is reexported by every backend _(the machinery behind-the-scenes making the figure)_, so you don't have to specifically install or import it.
```
Comment (storopoli) [inline contents/dataVis.md:70]: perfect!

File: contents/dataVis.md [hunk 3]
```diff
@@ -327,7 +324,7 @@ sco(s)
 
 Now, let's move on and do a plot with LaTeX strings and a custom theme.
 
-### LaTeXStrings
+### using LaTeXStrings
```
Comment (storopoli) [inline contents/dataVis.md:327]: I've forgotten our convention aren't we using `LaTeXStrings.jl` or `LaTeXStrings` for packages?

@rikhuijzer ?

Comment (rikhuijzer) [reply contents/dataVis.md:327]: With jl, but Lazaro removed it here so that's okay. I guess it looks better in the menu without jl

Comment (lazarusA) [reply contents/dataVis.md:327]: my bad. Yes with .jl, I will updated it.

File: contents/dataVis.md (hunk 1 above)
Comment (lazarusA) [reply contents/dataVis.md:64]: Full line would be, _Simon Danish, Creator of `Makie.jl`_ ?

Comment (storopoli) [reply contents/dataVis.md:64]: Yes. Or does he have a better title? Creator and lead developer? Creator and Maintainer?

**Pull request merged**